plugin/trino-hive/src/main/java/io/trino/plugin/hive/procedure/SyncPartitionMetadataProcedure.java

```diff
@@ -172,6 +172,9 @@ private static Set<String> doListPartitions(TrinoFileSystem fileSystem, Location
             if (!isValidPartitionPath(path, column, caseSensitive)) {
                 continue;
             }
+            if (!caseSensitive) {
+                path = path.toLowerCase(ENGLISH);
+            }
```

Comment:
Could you add a test to `TestHive3OnDataLake` class?

Reply:
This will lower case even the partition values. Thats not good.
 
Meaning it will come  off S3 as
indexTicker=LUTLTRUU Index/priceYear=2022/priceMonth=3
 
Your code is lower casing everything.
indexticker=lutltruu index/priceyear=2022/pricemonth=3

HIVE metastore stores it as
![image](https://github.com/trinodb/trino/assets/3594022/ba943b0b-6b8a-4ff4-9324-5a879d06ab47)

This wont work
Technically the comparizon should be a case insensitive comparison of the partition names. The values after the = are case sensitive and should not be changed.